docs/source/system-architecture.md

```diff
@@ -291,6 +299,18 @@ located.
 
 See [Enabling IBM Spectrum Conductor Support](kernel-conductor.html#enabling-ibm-spectrum-conductor-support) for details.
 
+###### CustomResourceProcessProxy
+Enterprise Gateway also provides a implementation of a process proxy derived from KubernetesProcessProxy 
+called CustomResourceProcessProxy. 
+
+Instead of creating kernel based on Kubernetes pod, CustomResourceProcessProxy
+manages kernel by implementation of custom resource definition(CRD), e.g. SparkApplication is a CRD which include
+plenty component of a Spark on Kubernetes application.
+
+If you want to include a new CRD, just wrt [spark_operator.py](https://github.com/jupyter/enterprise_gateway/blob/master/etc/kernel-launchers/kubernetes/scripts/launch_custom_resource.py)
```

Comment:
I don't understand the _just wrt_ portion of this statement.  _Just with respect to `spark_operator.py` and `spark_python_operator` kernelspec. _ doesn't really parse for me.  What are you trying to say about this?

Reply:
my fault

here, i should write down much more detail information about how to quickly extend the crd process proxy